flexinput/src/main/java/com/lytefast/flexinput/fragment/CameraFragment.kt

```diff
@@ -63,19 +63,20 @@ open class CameraFragment : PermissionsFragment() {
   private var flashState: Int
     @UiThread
     get() = with(PreferenceManager.getDefaultSharedPreferences(requireContext())) {
-      getInt(PREF_FLASH_STATE, CameraView.FLASH_AUTO)
+      getInt(PREF_FLASH_STATE, CameraKit.FLASH_AUTO)
     }
     @UiThread
-    set(@CameraView.Flash value) = with(PreferenceManager.getDefaultSharedPreferences(requireContext())) {
+    set(value) = with(PreferenceManager.getDefaultSharedPreferences(requireContext())) {
       edit()
           .putInt(PREF_FLASH_STATE, value)
           .apply()
     }
 
   var onFlashChanged: (flashBtn: View, newState: Int) -> Unit = { flashBtn, newState ->
     @StringRes val messageStringRes = when (newState) {
-      CameraView.FLASH_ON -> R.string.flash_on
-      CameraView.FLASH_OFF -> R.string.flash_off
+      CameraKit.FLASH_ON -> R.string.flash_on
+      CameraKit.FLASH_OFF -> R.string.flash_off
+//      CameraKit.FLASH_AUTO,
```

Comment:
Can delete?

Reply:
I want to leave this in so we know what `else` means. There's also another option which is `TORCH` which we don't cycle through.